### ISSUE TITLE
[Event Request] codeunit 410 "Update Analysis View" - OnAfterInitLastEntryNo

### ISSUE THREAD

**Issue by miljance** [microsoft/ALAppExtensions#19833]
I have an issue with the event OnAfterInitLastEntryNo in the codeunit 410 "Update Analysis View" as it is called only once after the function InitLastEntryNo is called. There are however two more calls of this function in the codeunit.
I have following suggestions. Any of those is fine

1. Please move the event OnAfterInitLastEntryNo to the function InitLastEntryNo or
2. Introduce the new one in the function InitLastEntryNo which will unfortunatelly have to have a different name or
3. Call the current OnAfterInitLastEntryNo after each call of the function InitLastEntryNo

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 47288.
